doc/design/configuration.md

```diff
@@ -0,0 +1,83 @@
+# Configuration
+
+Three primary elements make up a functioning VCH - these are referred to as components in the rest of this document, along with their subcomponents:
+
+1. [vic-machine](vic-machine.md) - the mechanism by which the VCH is deployed, configured, and inspected
+2. appliance - a VM that runs the VCH logic ([docker personality](docker-api-server.md), [log server](vicadmin.md), [port-layer](arch/vic-port-layer-overview.md), et al)
+3. containerVMs - VMs that _are_ containers ([tether](tether.md), and container process)
+
+This document discusses how configuration between components is done, and why it's done in that manner.
+
+## Component interactions
+
+For both the VCH and containerVMs the configuration is used for two way communication, with a component updating its own configuration as an asynch publishing mechanism for low-frequency, persistent information. The fields listed below are examples rather than comprehensive lists.
+
+### vic-machine configuration of appliance
+
+
+Configures VCH on initial deploy:
+* _vSphere target and credentials_ (hidden)
+* networks (external, client, management, bridge, mapped vsphere networks)
+* vsphere compute path (e.g. resource pool)
+* image store URI
+* container store URI
+* permissible URIs for volumes
+* white/black list of registries
+* _etc_
```

Comment:
Is this also where you expect mapping of externally scoped networks will happen?